package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@edozo/mechanical-wombat",
-  "version": "3.2.0",
+  "version": "3.2.1",
```

Comment:
Because I'm pedantic, I'd suggest this to be `3.3.0`, as it's an additional feature, but not a breaking change. So it's a minor change, not a patch. 
As per the [semantic versioning guide](https://semver.org/).